packages/block-editor/src/components/link-control/link-preview.js

```diff
@@ -41,7 +41,7 @@ export default function LinkPreview( {
 	const hasRichData = richData && Object.keys( richData ).length;
```

Comment:
Do we still need `hasRichData`? It adds an `.is-rich` class, but that class is unused in the CSS.

Reply:
Oh, looks like it was only ever used for tests. Not going to rip that out right now.